recipes/bempp-cl/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "bempp-cl" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bempp/bempp-cl/archive/v{{ version }}.tar.gz
+  sha256: 591817a2f724bfdda0e51073b99301db478090e778ab7b4d55f462e74b373c1b
+build:
+  number: 0
+  skip: True  # [py!=37]
+  skip: True  # [win]
```

Comment:
Is there a reason why windows is skipped?

Reply:
The pocl package is not available on Windows. It provides a CPU OpenCL driver. The package also works without pocl present if there is another OpenCL driver on the system (e.g. Intel, Nvidia), but requires at least one to function properly. Is there an option to include pocl only on non-windows builds? Then I could enable it.

Just for info. The Python version restriction is due to a known Numba bug with Python 3.8. I am waiting for the next Numba version to be released (should be quite soon), so that we can enable Python 3.8.

Reply:
One more comment though. If it is possible to enable pocl only conditionally depending on environment I would prefer to do this first for the next version of the library as currently it does not exit gracefully if no OpenCL driver is present, confusing users as to the nature of the error message.

Reply:
Thanks for clarifying that @tbetcke! In general, if you want to use `pocl` only for linux and osx, and not windows, then you can use:
`pocl  # [unix]`
in the `run` requirement.
I don't see any other OpenCL packages available for windows, so maybe skipping windows all together is the best approach.

Reply:
Thanks for the clarification. I recently talked to an Intel Developer if they could make their OpenCL drivers available through conda (freely available, but current license prohibits distribution that is not part of a larger package). He told me that he asked their legal team to look into this.

Reply:
That would be great, to create conda packages if the license allows distribution. In the meantime, skipping windows is ok.